### DISCUSSION TITLE
Nozzle to far from bed to even manually level

### DISCUSSION THREAD

**Discussion by Get0ffMyLawnV2** [Jyers/Marlin#512] [Q&A] [answered]
Been having trouble with Smith3d firmware so I 'finally' made the jump to Jyers.  Love the amount of options with this firmware but I'm a n00b....  4.2.7 board with printermods mdd, microswiss hotend, bltouch 3.1 and herome shourd (if this matters).  Fresh install of E3V2-UBL-BLTouch-10x10-HS-v4.2.7.bin went just fine.  So I'm starting from scratch with leveling and when I go to manually level the bed, my leveling knobs are maxed out and still can't get close enough to the nozzle so it needs to be lowered.  I've set my probe X/Y offsets appropriately based off the HeroMe documentation.  What do I need to do to lower the nozzle when I Auto Home so I can manually level this sucker?  Or is that the issue?  Leveling knobs about to fall right now it is that far away lol.  If this is not appropriate for here, again, I apologize.

**Comment by Jyers** [comment, marked as answer]
You need to set your Z offset accordingly. In the prepare menu there is a Z offset menu that makes this very easy. Just enable live adjust and lower the offset until it is at the desired height.
